data/lang/module-findperson/en.json

```diff
@@ -0,0 +1,554 @@
+  },
+  "COMPANY_2": {
+    "description": "Name of a company",
+    "message": "{name} Firearms Company"
+  },
+  "COMPANY_3": {
+    "description": "Name of a company",
+    "message": "{name} Fleet Systems Foundation"
+  },
+  "COMPANY_4": {
+    "description": "Name of a company",
+    "message": "{name} Defence Systems Company"
+  },
+  "COMPANY_5": {
+    "description": "Name of a company",
+    "message": "{name} Surveillance Tools Corporation"
+  },
+  "COMPANY_6": {
+    "description": "Name of a company",
+    "message": "{name} Special IT Services Foundation"
+  },
+  "DANGER": {
+    "description": "The risk level",
+    "message": "Danger:"
+  },
+  "DEADLINE": {
+    "description": "Must be delivered by",
+    "message": "Deadline:"
+  },
+  "DENY_1": {
+    "description": "",
+    "message": "Excuse me Sir, I need a pilot who has more experience."
+  },
+  "DENY_2": {
+    "description": "",
+    "message": "Sorry, I don't have time for you."
+  },
+  "DISTANCE": {
+    "description": "Distance in space, between two locations",
+    "message": "Distance:"
+  },
+  "FAILUREMSG_1": {
+    "description": "",
+    "message": "I'm not satisfied with your performance. I won't pay you!"
+  },
+  "FAILUREMSG_2": {
+    "description": "",
+    "message": "You are too late! Sorry, you won't get paid!"
+  },
+  "FIND_PERSON": {
+    "description": "Name of mission type",
+    "message": "Find person"
+  },
+  "GREETING_DELIVER_DOCUMENT": {
+    "description": "",
+    "message": "Damn, you got me. OK, I have signed the document. Please return it to {client}!"
+  },
+  "GREETING_DELIVER_MESSAGE": {
+    "description": "",
+    "message": "Thank you for the message. I have transmitted an answer for {client} to your ship's computer."
+  },
+  "GREETING_EVACUATION": {
+    "description": "",
+    "message": "Heaven, it's nice to see you! We can take off immediately. There's nothing keeping me here anymore."
+  },
+  "GREETING_PRIVATEER": {
+    "description": "",
+    "message": "Uh, you work for {client}? I have copied valuable data to your ship's computer. {client} will find this useful!"
+  },
+  "GREETING_TAXI": {
+    "description": "",
+    "message": "Oh, no. That's bad news. Let's fly back immediately."
+  },
+  "GREETING_TRAITOR": {
+    "description": "",
+    "message": "What? {client} wants a statement? OK, please return and hand over this document!"
+  },
+  "HOW_CAN_I_FIND_HER": {
+    "description": "",
+    "message": "How can I find her?"
+  },
+  "HOW_CAN_I_FIND_HIM": {
+    "description": "",
+    "message": "How can I find him?"
+  },
+  "HOW_MUCH_TIME": {
+    "description": "",
+    "message": "How much time do I have?"
+  },
+  "HOW_MUCH_TIME_DELIVER_DOCUMENT": {
+    "description": "",
+    "message": "Please be back by {date}."
+  },
+  "HOW_MUCH_TIME_DELIVER_MESSAGE": {
+    "description": "",
+    "message": "If I don't hear from you by {date}, I'm gonna hire another pilot!"
```

Comment:
"gonna" might sting in some eyes. Might be fine for very informal talk, but if conversation is in writing, e.g. through a BBS, I'd prefer "I will".

(I don't think we've decided if the BBS is a visi-phone system, or how it actually functions).